modules/virt-configuring-pod-log-verbosity.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/support/virt-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-pod-log-verbosity_{context}"]
+= Configuring {VirtProductName} pod log verbosity
+
+You can configure the verbosity level of {VirtProductName} pod logs by editing the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+. To set log verbosity for specific components, open the `HyperConverged` CR in your default text editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Set the log level for one or more components by editing the `spec.logVerbosityConfig` stanza. For example:
```

Comment:
```suggestion
. Set the log level for one or more components by editing the `spec.logVerbosityConfig` path. For example:
```

Reply:
This is incorrect. The correct term for a section in a YAML file is "stanza", not "path".